src/index.js

```diff
@@ -0,0 +1,182 @@
+const inquirer = require("inquirer");
+const fs = require("fs");
+
+//classes
+const Employee = require("./lib/Employee");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+
+//html templates go here
+const addManagerCard = require("./cards/card-manager");
+const addEngineerCard = require("./cards/card-engineer");
+const addInternCard = require("./cards/card-intern");
+const wrapProfileCards = require("./cards/card-wrapper");
+
+//team members array
+const team = [];
+
+const teamNameQuestion = [
+  {
+    name: "teamName",
+    type: "input",
+    message: "What is the name of your team?",
+  },
+];
+
+const managerQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Please enter the Manager's name:",
+  },
+  {
+    type: "input",
+    name: "id",
+    message: "Please enter the Manager's ID number:",
+  },
+  {
+    type: "input",
+    name: "email",
+    message: "Please enter the Manager's Email address:",
+  },
+  {
+    type: "input",
+    name: "officeNumber",
+    message: "Please enter the Manager's Office Number:",
+  },
+  {
+    type: "list",
+    name: "upNext",
+    message: "What would you like to do next?",
+    choices: ["Add Engineer", "Add Intern", "My team is complete!"],
+  },
+];
+
+const engineerQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Please enter Engineer's name:",
+  },
+  {
+    type: "input",
+    name: "id",
+    message: "Please enter Engineer's ID number:",
+  },
+  {
+    type: "input",
+    name: "email",
+    message: "Please enter the Engineer's Email address:",
+  },
+  {
+    type: "input",
+    name: "github",
+    message: "Please enter the Engineer's Github username:",
+  },
+  {
+    type: "list",
+    name: "upNext",
+    message: "What would you like to do next?",
+    choices: ["Add Engineer", "Add Intern", "My team is complete!"],
+  },
+];
+
+const internQuestions = [
+  {
+    type: "input",
+    name: "name",
+    message: "Please enter Intern's name:",
+  },
+  {
+    type: "input",
+    name: "id",
+    message: "Please enter Intern's ID number:",
+  },
+  {
+    type: "input",
+    name: "email",
+    message: "Please enter the Intern's Email address:",
+  },
+  {
+    type: "input",
+    name: "school",
+    message: "Please enter the Intern's school:",
+  },
+  {
+    type: "list",
+    name: "upNext",
+    message: "What would you like to do next?",
+    choices: ["Add Engineer", "Add Intern", "My team is complete!"],
+  },
+];
+
+const askQuestions = async (questionArr) => {
```

Comment:
You're using a mixture of `async` code here but also the older `.then()` syntax. Try to rewrite this just using `async` `await`. It'd look sometihng like this:

```
const member = await inquirer.prompt(questionArr)
team.push(member)
```
Instead of using `.catch()`, we could wrap the whole thing in a `try` `catch` block